photon-lib/py/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+import subprocess
+
+versionStr = subprocess.check_output(['git', 'describe', '--tags', "--match=v*", "--always"]).decode('utf-8').strip()
+print(f"Building version {versionStr}")
+
+setup(
+    name='photonlibpy',
```

Comment:
Consider using pyproject.toml, or setup.cfg

Reply:
Given that setup.py has expanded already to have build-time python logic.... I'm not sure I know enough to migrate it quickly? As long as you're not certain this will cause immedeate issue, I'd personally rather stick with "works" and enter an issue to followup next summer?